src/librustdoc/html/render/print_item.rs

```diff
@@ -958,6 +958,20 @@ fn item_trait(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, t: &clean:
     let cloned_shared = Rc::clone(&cx.shared);
     let cache = &cloned_shared.cache;
     let mut extern_crates = FxHashSet::default();
+
+    if !t.is_object_safe(cx.tcx()) {
```

Comment:
Shouldn't the condition be the opposite? Don't we want to display this information in case the trait is object safe? Or did I miss something?

Reply:
Ok nevermind. Just realized it were displaying the opposite information.